package.json

```diff
@@ -27,7 +27,7 @@
     "node.bittrex.api": "^0.2.5",
     "number-abbreviate": "^2.0.0",
     "numbro": "git+https://github.com/carlos8f/numbro.git",
-    "poloniex.js": "0.0.7",
+    "poloniex.js": "git+https://github.com/premasagar/poloniex.js.git",
```

Comment:
Sorry about that. I'll look into this later tonight or tomorrow. I would love to get this all shaped up and glad we could share ideas here.
Had a close family member die last week, pretty rough, but what can you do. I won't be active as much, but will try my best to fix this up for you.